spring-cloud-gcp-data-firestore/src/main/java/com/google/cloud/spring/data/firestore/FirestoreTemplate.java

```diff
@@ -62,61 +65,75 @@
  * @since 1.2
  */
 public class FirestoreTemplate implements FirestoreReactiveOperations {
-	private FirestoreClassMapper classMapper;
 
 	private static final int FIRESTORE_WRITE_MAX_SIZE = 500;
 
-	private static final String NAME_FIELD = "__name__";
+	private static final String NOT_FOUND_DOCUMENT_MESSAGE = "NOT_FOUND: Document";
 
-	private static final StructuredQuery.Projection ID_PROJECTION = StructuredQuery.Projection.newBuilder()
-			.addFields(StructuredQuery.FieldReference.newBuilder().setFieldPath(NAME_FIELD).build())
-			.build();
+	private final String nameField = FieldPath.documentId().toString();
```

Comment:
Why not `static`?

Reply:
Ah `static` makes sense. Also reverted some of the unnecessary changes I made to this file.